science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+### 2.1 RAT
+
+In total, there were approximately 27 million reported rapid antigen test results (RAT), of which approximately 7.8 million were preceded by at least one risk exposure. The following figure shows the progression of positivity rates over time for each type of risk notification. Rapid antigen test results have only been registered via data donation since 24 June 2021.
+
+<figure class="figure-img">
+<img src="./ppa_risk_infection_ct_timeline_rat-1_en.png" class="figure-img img-fluid" alt="Figure 1: Positivity rate after risk notification over time (RAT)."/>
+<figcaption aria-hidden="true">Figure 1: Positivity rate after risk notification over time (RAT).</figcaption>
+</figure>
+
+Here, the positivity rate at increased risk is almost always higher than that at lower risk with or without a risk exposure. 
+
+The positivity rate also varies here depending on the infection event and warning. After a red warning, the positivity rate is between 0 and 0.25, for a lower risk with risk exposure between 0 and 0.16 and for a lower risk without risk exposure between 0 and 0.15.
+
+### 2.2 PCR
+
+In total, there were approximately 9.8 million reported PCR test results (PCR), of which approximately 2.7 million were preceded by at least one risk exposure. The following figure shows the progression of positivity rates over time for each type of risk notification. PCR test results have been registered via data donation since 10 March 2021.
+
+<figure class="figure-img">
+<img src="./ppa_risk_infection_ct_timeline_pcr-1_en.png" class="figure-img img-fluid" alt="Figure 2: Positivity rate after risk notification over time (PCR)."/>
+<figcaption aria-hidden="true">Figure 2: Positivity rate after risk notification over time (PCR).</figcaption>
+</figure>
+
+What is striking here is the significantly greater positivity rate for increased risk compared to that for lower risk without a risk exposure in times of relatively lower incidence. In times of high incidence, the curves run almost the same. Below we will look at this relationship in more detail.
+
+The positivity rate also varies here depending on the infection event and warning. After a red warning, the positivity rate is between 0 and 0.88, for a lower risk with risk exposure between 0 and 0.8 and for a low risk without risk exposure between 0 and 0.78.
+
+The [RKI‑test figures](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Daten/Testzahlen-gesamt.html) are given for comparison. The somewhat greater proportion of persons tested positive in the CWA data donation is due to the different underlying survey population. For example, people under the age of 16 are not formally allowed to use the Corona‑Warn‑App and are therefore under-represented in its user base. On the other hand, the RKI figures also increasingly include routine and pooling test results, which are probably rarely, if ever registered via the Corona‑Warn‑App and, in accordance with their purpose, are more often negative.
```

Comment:
Add "(German)" for RKI figures, since they are not available in English.

Reply:
right, will add (German)